cmd/dist/dist.go

```diff
@@ -19,9 +21,12 @@ import (
 )
 
 var (
-	synologyPackageCenter bool
-	qnapPrivateKeyPath    string
-	qnapCertificatePath   string
+	synologyPackageCenter   bool
+	gcloudCredentialsBase64 string
+	gcloudProject           string
+	gcloudKeyring           string
```

Comment:
nit: can we prefix these with `qnap` too, to make it clearer what these vars are for?

Reply:
I intentionally used gcloud because in the future, I hope to use these for other platforms (i.e. Windows) as well.